verify/verifier.go

```diff
@@ -25,6 +26,14 @@ const (
 )
 
 type Config struct {
+	// Allowed distance (in L1 blocks) between the eigenDA reference block number (RBN) of the batch the blob is included in,
+	// and the L1 block number at which the blob cert was included in the batcher's inbox.
+	// If batch.RBN + RollupBlobInclusionWindow < cert.L1InclusionBlock, the batch is considered stale and verification will fail.
+	// This check is optional and will be skipped when RollupBlobInclusionWindow is set to 0.
+	RollupBlobInclusionWindow uint32
```

Comment:
We need to figure out how to manage this securely. See discussion in https://github.com/Layr-Labs/eigenda-proxy/pull/226#discussion_r1908276688